blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequest.java

```diff
@@ -148,6 +148,13 @@ public BluePipeline create(Reachable parent) throws IOException {
                         }
                     }
 
+                    // Add any existing discovered repos
```

Comment:
Without this, if you switch from "discover all" to "add a single pipeline" - it actually wipes out all other discovered pipelines. This simply iterates through the existent ones, and ensures they end up added to the pattern expression - this matches up with the description on the GUI which is the correct design intention.